<launch>
	<rosparam command="load" file="$(find grasper)/launch/hotStab.yaml" />
	 <arg name="joint_state" default="/arm5e/joint_state_angle" />
	 <arg name="joint_state_command" default="/arme5e/command_angle" />
	 <arg name="joint_state_fixed" default="/arm5e/joint_state_angle_fixed" />
	 <arg name="detector_start" default="/template_pose/start_detection" />
	 <arg name="detector_stop" default="/template_pose/stop_detection" />
	 <arg name="max_current" default="1.6" />
	 <arg name="velocity_aperture" default="0.4" />
	 <arg name="gripper_manipulation" default="1.0" />
	 <arg name="gripper_closed" default="0.0" />
		 
	 <param name="joint_state" value="$(arg joint_state)" />
	 <param name="joint_state_command" value="$(arg joint_state_command)" />
	 <param name="joint_state_fixed" value="$(arg joint_state_fixed)" />
	 <param name="detector_start" value="$(arg detector_start)" />
	 <param name="detector_stop" value="$(arg detector_stop)" />
	 <param name="max_current" value="$(arg max_current)" />
	 <param name="velocity_aperture" value="$(arg velocity_aperture)" />
	 <param name="gripper_manipulation" value="$(arg gripper_manipulation)" />
	 <param name="gripper_opened" value="$(arg gripper_closed)" />

<node name="hotStab" pkg="grasper" type="hot_stab_server" respawn="false" output="screen" />

</launch>